package.json

```
@@ -100,7 +100,7 @@
     "@walletconnect/jsonrpc-utils": "^1.0.8",
     "@zeriontech/components": "^1.0.3",
     "@zeriontech/hardware-wallet-connection": "^1.0.11",
-    "@zeriontech/transactions": "^0.0.14",
+    "@zeriontech/transactions": "file:../transactions-ui/zeriontech-transactions-0.0.14.tgz",
```

```suggestion
    "@zeriontech/transactions": "^0.0.14",
```
I assume you used that package from fs temporarily